src/adm/settings.rs

```diff
@@ -333,70 +395,54 @@ impl AdmFilterSettings {
 /// This allows `CONTILE_ADM_SETTINGS` to either be specified as inline JSON, or if the
 /// Settings are too large to fit into an ENV string, specified in a path to where the
 /// settings more comfortably fit.
-impl TryFrom<&mut Settings> for AdmFilterSettings {
-    type Error = ConfigError;
-
-    fn try_from(settings: &mut Settings) -> Result<Self, Self::Error> {
-        // TODO: Convert these to macros.
-        if settings.adm_sub1.is_none() {
-            return Err(ConfigError::Message(format!(
-                "Missing argument {}",
-                "adm_sub1"
-            )));
-        }
-        if settings.adm_partner_id.is_none() {
-            return Err(ConfigError::Message(format!(
-                "Missing argument {}",
-                "adm_partner_id"
-            )));
-        }
-        if settings.adm_settings.is_empty() {
-            return Ok(Self::default());
-        }
-        let mut settings_str = settings.adm_settings.clone();
-        if Path::new(&settings_str).exists() {
-            if let Ok(mut f) = File::open(&settings.adm_settings) {
-                settings_str = String::new();
-                f.read_to_string(&mut settings_str).map_err(|e| {
-                    ConfigError::Message(format!(
-                        "Could not read {}: {:?}",
-                        settings.adm_settings, e
-                    ))
-                })?;
-            }
-        }
-        AdmFilterSettings::try_from(settings_str)
-    }
-}
 
 /// Construct the AdmFilter from the provided settings.
 ///
 /// This uses a JSON construct of settings, e.g.
 /// ```javascript
 /// /* for the Example Co advertiser... */
 /// {"Example": {
```

Comment:
Again, let's make this comment match what we're doing, so add in the parent `adm_advertisers` key.